spec/integration/active_record_compatibility_spec.rb

```diff
@@ -6,7 +6,7 @@
   include Helpers::Plugins
   include Helpers::Translates
   # Enable all plugins that are enabled by default pre v1.0
-  plugins :active_record, :reader, :writer, :cache, :dirty, :presence, :query, :attribute_methods
+  plugins :active_record, :reader, :writer, :cache, :dirty, :presence, :query, :attribute_methods, :fallbacks
```

Comment:
Interestingly, this PR highlighted that the spec here was not actually enabling the `fallbacks` plugin, although specs were testing fallbacks stuff. So basically those tests were doing nothing useful :facepalm: